packages/sessions-sdk-react/src/components/session-panel.tsx

```diff
@@ -208,6 +208,7 @@ const Tokens = ({
               icon: token.image,
               symbol: token.symbol,
               tokenName: token.name,
+              price: token.price,
```

Comment:
Remove all changes in this file since we no longer need to thread `price` through as props